test/cpp/api/modules.cpp

```diff
@@ -1868,6 +1884,12 @@ TEST_F(ModulesTest, PrettyPrintSoftMarginLoss) {
   ASSERT_EQ(c10::str(SoftMarginLoss()), "torch::nn::SoftMarginLoss()");
 }
 
+TEST_F(ModulesTest, PrettyPrintLoss) {
+  ASSERT_EQ(
+      c10::str(NLLLoss(NLLLossOptions().ignore_index(-100))),
+      "torch::nn::NLLLoss(ignore_index=-100)");
```

Comment:
I believe in Python version it doesn't print any options arguments, and we can simplify this to just check for `"torch::nn::NLLLoss()"` here :D